package.json

```
@@ -40,15 +40,15 @@
     "eslint-plugin-jest": "^22.6.4",
     "eslint-plugin-prettier": "^3.1.0",
     "execa": "^1.0.0",
-    "jest": "^21.0.0 || ^22.0.0 || ^23.0.0 || ^24.0.0",
+    "jest": "^21.0.0 || ^22.0.0 || ^23.0.0 || ^24.0.0 || ^25.0.0",
```

```suggestion
    "jest": "^21.0.0 || ^22.0.0 || ^23.0.0 || ^24.0.0 || ^25.1.0",
```

25.0.0 is a beta version released back in the summer, unfortunately as version `25.0.0` (never tagged as `latest` though). 25.1.0 is the first stable release of 25

---

Good to know! Will update.